test/integration/targets/ios_bgp/tests/cli/basic.yaml

```diff
@@ -86,6 +86,11 @@
             remote_as: 64511
             description: EBGP_NBR_1
             local_as: 64497
+
+          - neighbor: 10.10.20.20
+            remote_as: 65012
+            description: EBGP_NBR_2
+            ebgp_multihop: 100
```

Comment:
You've added something to the config, but are not asserting that it exists in the output. Please either remove the config or update the assert to validate the result

Reply:
I needed to add a bgp  neighbor outside adress-family ipv4. Moving down that config to the relevant test.